slo/rules.go

```diff
@@ -712,7 +712,7 @@ func (o Objective) IncreaseRules() (monitoringv1.RuleGroup, error) {
 			}.replace(expr)
 
 			rules = append(rules, monitoringv1.Rule{
-				Alert: "SLOMetricAbsent",
+				Alert: o.AlertNameMetricAbsent(),
```

Comment:
This is one of many places that need to be updated. 
Searching for `SLOMetricAbsent` will show the others.